documentation/modules/upgrading/con-upgrade-strategies-for-upgrading-clients.adoc

```diff
@@ -6,65 +6,43 @@
 
 = Strategies for upgrading clients
 
-The right approach to upgrading your client applications (including Kafka Connect connectors) depends on your particular circumstances.
+[role="_abstract"]
+The goal of upgrading clients is to enable them to consume new message formats produced by updated producers.
```

Comment:
To me, you upgrade clients to benefits from new features, fixes and improvements.

Reply:
Yeah. Let's get away from strategy here.
Updated as follows:

> Upgrading Kafka clients ensures that they benefit from the features, fixes, and improvements that are introduced in new versions of Kafka. 
>Upgraded clients maintain compatibility with other upgraded Kafka components.
> The performance and stability of the clients might also be improved.